app/initializers/ember-cli-mirage.js

```diff
@@ -16,6 +16,8 @@ const { default: baseConfig, testConfig, makeServer } = config;
 export default {
   name: 'ember-cli-mirage',
   initialize(application) {
+    window.__app_for_mirage = application;
```

Comment:
@kiwi-josh minor, I wonder if it might be safer to use a `Symbol` here, to ensure no one else uses `__app_for_mirage` even accidentally

Reply:
@SergeAstapov yeah good call out - I've updated the code to use a `Symbol` instead. Let me know if theres a better way you would like it done.